### DISCUSSION TITLE
Help message can't be translated using gettext (string not marked with gettext) rich_utils.py:694

### DISCUSSION THREAD

**Discussion by johndoe46** [fastapi/typer#883] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# (generic, does not apply)
```

### Description

`rich_utils.py` has a useful message that suggests to try a command for help. However, this can't be translated using the usual gettext module because the string is not marked:

```
    if ctx is not None and ctx.command.get_help_option(ctx) is not None:
        console.print(
            f"Try [blue]'{ctx.command_path} {ctx.help_option_names[0]}'[/] for help.",
            style=STYLE_ERRORS_SUGGESTION,
        )
```

SInce the module already use gettext for other strings, it would be nice if this one could be included as well.

### Operating System

Linux, Windows, macOS, Other

### Typer Version

0.12.3

### Python Version

3.12.3

**Comment by svlandeg** [comment, marked as answer]
Makes sense to me! I proposed a PR here: https://github.com/tiangolo/typer/pull/886

**Reply by johndoe46** [reply]
Thanks ! I didn't know if I could propose a PR but yours looks perfect anyway
